ecc_enclave/enclave/shim.cpp

```diff
@@ -40,8 +40,8 @@ void get_creator_name(
 void get_state(
     const char* key, uint8_t* val, uint32_t max_val_len, uint32_t* val_len, shim_ctx_ptr_t ctx)
 {
-    uint8_t encoded_cipher[(max_val_len + SGX_AESGCM_IV_SIZE + SGX_AESGCM_MAC_SIZE + 2) / 3 * 4];
     uint32_t encoded_cipher_len;
+    uint8_t encoded_cipher[(max_val_len + SGX_AESGCM_IV_SIZE + SGX_AESGCM_MAC_SIZE + 2) / 3 * 4];
```

Comment:
any deeper reason why you swapped the order of the two vars?

Reply:
no, left over!

Reply:
well, spurious (non-)changes are a bit of a distraction ...